core/federated/RTI/rti_lib.c

```diff
@@ -1997,6 +1997,11 @@ int process_args(int argc, const char* argv[]) {
            i++;
            i += process_clock_sync_args((argc-i), &argv[i]);
         } else if (strcmp(argv[i], "-a") == 0 || strcmp(argv[i], "--auth") == 0) {
+            #ifndef __RTI_AUTH__
+            fprintf(stderr, "Error: --auth requires the RTI to be built with the -DAUTH=ON option.\n");
+            usage(argc, argv);
+            return 0;
```

Comment:
Shouldn't this exit with a non-zero value? Otherwise, tests will pass?

Reply:
This return value is actually not the exit code.
The return value is checked here: https://github.com/lf-lang/reactor-c/blob/main/core/federated/RTI/rti.c#L65
And return value zero actually leads to a non-zero exit code.

Reply:
As I mentioned above, the return value is not an exit code.